pyanaconda/storage_utils.py

```diff
@@ -47,7 +47,9 @@
 from pykickstart.constants import AUTOPART_TYPE_LVM, AUTOPART_TYPE_LVM_THINP
 
 import logging
```

Comment:
It would be nice to have all the logging logic in one file, so we wouldn't need to import the `logging` anymore.